infocmdb/ci_type.go

```diff
@@ -93,3 +94,38 @@ func (c *Client) GetCiTypeName(ciId int) (ciTypeName string, err error) {
 
 	return
 }
+
+type respSetTypeOfCi struct {
+	Message bool `json:"success"`
+}
+
+func (c *Client) SetTypeOfCi(ciId int, ciType string) (respStatus string, err error) {
```

Comment:
Is the respStatus return value useful for the caller?
If it serves no purpose I'd rather like to get rid of it.

Reply:
respStatus is removed in last commit